src/features/bookings/components/BookingDetailsTicketContent.tsx

```diff
@@ -107,11 +151,19 @@ const Title = styled(Typo.Title3).attrs(getHeadingAttrs(1))({
   maxWidth: '100%',
 })
 
-const TicketInnerContent = styled.View(({ theme }) => ({
-  justifyContent: 'center',
+const TicketInnerContent = styled.View({
+  paddingTop: getSpacing(6),
+})
+
+const TicketInfo = styled(Typo.Body).attrs(getHeadingAttrs(1))({
+  textAlign: 'center',
+  maxWidth: '100%',
```

Comment:
Tu peux enlever le `getHeadingAttrs` ici
```suggestion
const TicketInfo = styled(Typo.Body)({
  textAlign: 'center',
  maxWidth: '100%',
```